### PR TITLE
UGDG add-ons

### PR DESCRIPTION
**Pull request by russelltankaimin**
# WORK TODO
- [x] Prefix (UG)
- [x] Person's attributes (DG)
- [x] Timetable (DG)

## Request:
Please click on the 2 pages and have a scroll through. If there are any structural changes. Indicate request changes or ping me directly here. Thanks.

View page [DG](https://russelltankaimin.github.io/tp/DeveloperGuide.html) and [UG](https://russelltankaimin.github.io/tp/UserGuide.html)

### CODE REVIEW

**Review by kennycjy — APPROVED** [review]
Other than what I commented. LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -666,6 +667,16 @@ If you want to end the application, simply type `exit`, or click on the `Exit` b
...
 ### Prefixes
 
+| Prefix | Representation  |
+|--------|-----------------|
+| n/     | Name            |
+| a/     | Address         | 
+| p/     | Telegram Handle | 
+| e/     | Email           | 
+| g/     | Group           | 
+| m/     | Module          |
```
[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:677]: This is strictly just for `Prefix`. If there are no `Prefix` for lessons, then it should not be inside.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -392,13 +392,67 @@ The `Model` component,
...
+
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning**:
+Group Tag Set must contain unique group names.
+
+</div>
+
 #### **Module Tag Set**
 
+Represents a collection of modules that the user or contact is enrolled into.
```
[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/DeveloperGuide.md:448]: No. It should be under `Commitment`

File: docs/DeveloperGuide.md
Comment (seanfirefox) [inline docs/DeveloperGuide.md:1]: LGTM for the DG.

File: docs/images/ViewActivityDiagram.svg
Comment (seanfirefox) [inline docs/images/ViewActivityDiagram.svg:1]: This looks a lot more comprehensive than the previous activity diagram.

File: docs/UserGuide.md
Comment (seanfirefox) [inline docs/UserGuide.md:1]: LGTM for the UG too.

**Review by seanfirefox — APPROVED** [review]
LGTM!

**Pull request merged**